app/containers/CustomHeader/components/HeaderContainer/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useWindowDimensions, View, ViewProps, StyleSheet } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { useTheme } from '../../../../theme';
+
+interface IHeaderContainer extends ViewProps {
+	addExtraNotchPadding?: boolean;
+	isMasterDetail?: boolean;
+	customLeftIcon?: boolean;
+	customRightIcon?: boolean;
+}
+
+const HeaderContainer = (props: IHeaderContainer) => {
+	const insets = useSafeAreaInsets();
+	const { colors } = useTheme();
+	const { height, width } = useWindowDimensions();
+	const isPortrait = height > width;
+	const paddingBottom = isPortrait ? 12 : 4;
+	const statusBarPadding = props.addExtraNotchPadding ? insets.top : 0;
+	const paddingRight = props.isMasterDetail || !props.customRightIcon ? 4 : 16;
+
+	return (
+		<View
+			{...props}
+			style={{
+				alignItems: 'center',
+				flexDirection: 'row',
+				paddingBottom,
+				paddingTop: statusBarPadding + paddingBottom,
```

Comment:
What's the component height on iOS portrait 1x?
It looks like the padding could be a lot smaller and we would keep a more common UX with the height between 44~48.
Compared with some other apps like Whatsapp and Telegram and theirs is smaller.

Padding on 12 is too much. It could be 8 or less. We don't need multiple of 4s to make this one right.